tasks/mBTC.ts

```diff
@@ -60,9 +60,7 @@ task("mBTC-snap", "Get the latest data from the mBTC contracts")
         const { ethers, network } = hre
 
         let exposedValidator
-        if (network.name !== "mainnet") {
```

Comment:
Should we improve `getChain` to include `hardhat` and avoid using the `network.name` to be consistent?